lib/src/client.dart

```diff
@@ -2153,6 +2158,7 @@ class Client extends MatrixApi {
           await _handleRoomEvents(room, state, EventUpdateType.inviteState);
         }
       }
+      await database?.storeRoomUpdate(id, syncRoomUpdate, room.lastEvent, this);
```

Comment:
because the lastEvent of the room is calculated in the _handleRoomEvents method. If I want to keep this method where it was before, I would had to duplicate the logic to update the lastEvent. So... lazyness leaded to this :D

Reply:
I thought some of the _handleRoomEvents functions depended on the data already being persisted, but I guess that wasn't the case then! Thanks!